---
layout: page
title: Call for Applications | MSc Thesis Topic

---

|--------------|----------------------------------------
| **Title**    | Evaluation of genome-based growth predictions in _Microbacterium_
| **Advisor**  | Dr. Luis M. Rodriguez-R, Department of Microbiology and Digital Science Center (DiSC)
| **Co-advisor** | Dr. Sigrid Neuhauser, Department of Microbiology
| **Number of students** | 1
| **Language** | English

## Description

It has been previously shown that both the maximum growth rate and the optimal
growth temperature of bacteria imprints statistical signatures in their genomes.
Specifically, the codon bias of highly-transcribed genes such as ribosomal genes
with respect to the average codon usage of the rest of the genome appears to
predict the minimum doubling time of bacteria. We have generated a large
collection of growth curves from isolates of different species in the
genus _Microbacterium_, using controlled, identical conditions, and have
determined their corresponding genome sequences. The selected student will have
access to this data, and will evaluate the fit of growth curves to determine
both the minimum doubling time and the range of incertitude. Next, the student
will generate growth predictions on the basis of codon usage biases with
existing statistical models, and will provide a systematic evaluation of their
performance in this collection. Finally, the student will be able to explore the
functional annotation and other genomic signatures in order to identify
additional sources of information that could explain the prediction residuals,
potentially improving the growth rate predictions in other taxa.

## Prerequisites

Ongoing Master's degree in microbiology, a capacity for both independent and
team work, and interest in both molecular biology and genomics.

## Theoretical skills

Foundations on microbiology and molecular biology are highly desirable. Interest
in environmental genomics is required, but no previous knowledge necessary.
Strong foundations in statistics are highly appreciated.

## Practical skills

Basic knowledge on R and Bash are preferred, but not required.

